csrc/executor_utils.cpp

```diff
@@ -843,7 +843,7 @@ class NvrtcCompileDriver {
     char* log_buf = log_backing_buf.data();
     NVFUSER_NVRTC_SAFE_CALL(nvrtcGetProgramLog(program, log_buf));
     if (result != NVRTC_SUCCESS) {
-      NVF_ERROR(false, src, "\nCUDA NVRTC compile error: ", log_buf);
+      NVF_ERROR(false, "\nCUDA NVRTC compile error: ", log_buf);
```

Comment:
Does it make sense to do a regex and trim anything before `__global__`? still would like to have some quick look at the error log...

Reply:
Ah good idea. Let me try that.

Reply:
Good suggestion! Printout:
```
Traceback (most recent call last):
  File "/opt/pytorch/nvfuser/repro1541.py", line 16, in <module>
    fd.execute(inputs)
  File "/opt/pytorch/nvfuser/nvfuser/__init__.py", line 137, in execute
    result = self._execute(
RuntimeError: false INTERNAL ASSERT FAILED at "/opt/pytorch/nvfuser/csrc/executor_utils.cpp":847, please report a bug with repro script to NVFuser at https://github.com/NVIDIA/Fuser/issues. 
__global__ void nvfuser_pointwise_f0_c1_r0_g0(Tensor<__bfloat, 1, 1> T0, Tensor<__bfloat, 1, 1> T1) {
  nvfuser_index_t i0;
  i0 = ((nvfuser_index_t)threadIdx.x) + (128LL * ((nvfuser_index_t)blockIdx.x));
  if ((i0 < T0.logical_size[0LL])) {
    __bfloat T3[1];
    T3[0] = 0;
    T3[0]
       = T0[i0];
    __bfloat T4[1];
    T4[0]
       = -T3[0];
    T1[i0]
       = T4[0];
  }
}
}

CUDA NVRTC compile error: __tmp_kernel_pointwise_f0_c1_r0_g0.cu(9455): error: no operator "-" matches these operands
            operand types are: - <unnamed>::__bfloat
         = -T3[0];
           ^

1 error detected in the compilation of "__tmp_kernel_pointwise_f0_c1_r0_g0.cu".

Exception raised from invoke at /opt/pytorch/nvfuser/csrc/executor_utils.cpp:847 (most recent call first):
frame #0: nvfuser::nvfCheckFail(char const*, char const*, unsigned int, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) + 0xf3 (0x7f5df37f0d37 in /opt/pytorch/nvfuser/nvfuser/lib/libnvfuser_codegen.so)
frame #1: nvfuser::nvfErrorFail(char const*, char const*, unsigned int, char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) + 0x53 (0x7f5df39445f3 in /opt/pytorch/nvfuser/nvfuser/lib/libnvfuser_codegen.so)
frame #2: nvfuser::executor_utils::getCompiledKernel(std::optional<std::reference_wrapper<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, nvfuser::CompileParams const&, std::optional<long>) + 0x2699 (0x7f5df399a359 in /opt/pytorch/nvfuser/nvfuser/lib/libnvfuser_codegen.so)
frame #3: nvfuser::FusionExecutor::compileFusion(nvfuser::Fusion*, nvfuser::KernelArgumentHolder const&, nvfuser::LaunchParams const&, nvfuser::CompileParams, nvfuser::ScheduleHeuristic, long, long, long, long) + 0xea6 (0x7f5df395bf86 in /opt/pytorch/nvfuser/nvfuser/lib/libnvfuser_codegen.so)
frame #4: nvfuser::FusionKernelRuntime::compileKernel(nvfuser::KernelArgumentHolder const&, nvfuser::SegmentedGroup*) + 0x1c4 (0x7f5df3aac564 in /opt/pytorch/nvfuser/nvfuser/lib/libnvfuser_codegen.so)
frame #5: nvfuser::FusionKernelRuntime::compileFusionParallel(nvfuser::KernelArgumentHolder) + 0x41c (0x7f5df3ab25dc in /opt/pytorch/nvfuser/nvfuser/lib/libnvfuser_codegen.so)
frame #6: nvfuser::FusionExecutorCache::runFusionWithInputs(c10::ArrayRef<c10::IValue> const&, std::optional<nvfuser::PrimDataType>, std::optional<signed char>) + 0xa43 (0x7f5df3abdf63 in /opt/pytorch/nvfuser/nvfuser/lib/libnvfuser_codegen.so)
frame #7: nvfuser::python_frontend::FusionDefinition::execute(c10::ArrayRef<c10::IValue> const&, bool, bool, std::optional<signed char>) const + 0x16c (0x7f5df3dd522c in /opt/pytorch/nvfuser/nvfuser/lib/libnvfuser_codegen.so)
frame #8: <unknown function> + 0xf151e (0x7f5df412651e in /opt/pytorch/nvfuser/nvfuser/_C.cpython-310-x86_64-linux-gnu.so)
frame #9: <unknown function> + 0x177458 (0x7f5df41ac458 in /opt/pytorch/nvfuser/nvfuser/_C.cpython-310-x86_64-linux-gnu.so)
<omitting python frames>
frame #25: <unknown function> + 0x29d90 (0x7f5f72c2ed90 in /usr/lib/x86_64-linux-gnu/libc.so.6)
frame #26: __libc_start_main + 0x80 (0x7f5f72c2ee40 in /usr/lib/x86_64-linux-gnu/libc.so.6)
```